api/v1/app.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/python3
+"""
+Module status
```

Comment:
Module status?? This script is general for the next methods, please make a better documentation